Mlem/App/Models/Action/ActionAppearance+StaticValues.swift

```diff
@@ -279,4 +279,31 @@ extension ActionAppearance {
     static func viewVotes() -> Self {
         .init(label: "View Votes", color: .themedAccent, icon: Icons.votes)
     }
+        
+    static func collapse() -> Self {
+        .init(
+            label: "Collapse",
+            color: .themedColorfulAccent(4),
+            icon: Icons.collapse,
+            swipeIcon2: Icons.collapseFill
```

Comment:
In the swipe action settings, the icons still have squares around them:

<img src="https://github.com/user-attachments/assets/d332401e-07dc-47ea-aa6a-914ac75eafe8" width=50%>

I think we shouldn't have squared icons in the settings page - this can be done by writing something like:

```suggestion
            icon: Icons.collapse,
            swipeAction1: Icons.collapseSquare,
            swipeIcon2: Icons.collapseSquareFill
```

```swift
  // Icons
  static let collapse: String = "minus"
  static let collapse: String = "minus.rectangle"
  static let collapseFill: String = "minus.rectangle.fill"
```